src/xdocs/contributing.xml

```diff
@@ -83,15 +83,15 @@
         do take this one step further and measure the coverage of our
         unit tests using
 
-        <a href="http://cobertura.github.io/cobertura/">Cobertura</a>.
+        <a href="http://www.eclemma.org/jacoco/">JaCoCo</a>.
 
         This means it is not sufficient to pass all tests, but the tests
         should ideally execute each line in the code, code coverage should be 100%. To generate the
-        Cobertura report, run the Maven command
+        JaCoCo report, run the Maven command
 
-        <code>mvn cobertura:cobertura</code>.
+        <code>mvn test jacoco:restore-instrumented-classes jacoco:report@default-report</code>.
 
```

Comment:
Still not ideal. Perhaps some bash script may help.

Reply:
it is more then enough for now. We will improve it some time later on.